plugins/SLURM-Integration/install.sh

```diff
@@ -64,6 +64,19 @@ function update_slurm_conf()
     fi
 }
 
+function update_python_path_for_ufm_prolog_epilog()
+{
+    python_path=$(which python3)
```

Comment:
1. use ```type python3```, as ```which``` command is external and not mandatory. ```type``` command is shell built-in command which is always available in any distro
2. what if ```which python3``` fails to find python3? dont see proper error flow.
in line 71-72 the check and err message are wrong:

need to check that ```which python3``` succeeded to run
if succeeded to run - check that command result is executable file
replace with sed and only then give error that failed to replace (also check that conf file is writable)
think of all what can go wrong and help user to navigate